src/qualitycontrol.jl

```diff
@@ -58,20 +64,30 @@ function have_ok_coordinates(o::GBIFRecord)
 end
 
 """
-**Cleans a search output**
+**Filters a series of records on a single criteria**
+
+This function will take the filter function `f` and use it to *mask* the records
+that do not satisfy it. Note that if a record is *already* masked due to the
+application of a previous filter, its status will *not* be modified. The
+application of filters is therefore cumulative.
 
-This function loops through all records, and applies the `filters` to it. Filters
-are built-in or user-defined functions that return `true` when the record
-needs to be kept, and `false` when it needs to be discarded.
+To ensure consistency with the `filter!` function from Base, this function only
+accepts a *single* filter `f` at a time. To apply multiple filters, it is
```

Comment:
You can define `broadcastable` for `GBIFRecords`- then you can do `filter!.(vec_of_functions, myGBIFRecord)`

Reply:
Do you have examples of that?

Reply:
```julia
julia> struct MyType end

julia> myfun(f, x::MyType) = f(1)
myfun (generic function with 1 method)

julia> Base.broadcastable(x::MyType) = Ref(x)

julia> myfun.([rand, log, sin], mt)
3-element Array{Any,1}:
  [0.929257]
 0.0
 0.8414709848078965
```

Reply:
Amazing, thanks!